proposals/nnnn-safe-shared-contiguous-storage.md

```diff
@@ -177,6 +201,14 @@ extension UnsafeMutableRawBufferPointer: ContiguousStorage<UInt8> {
 }
 ```
 
+**TODO**: What is the `@_unsafeNonescapableResult` annotation? Would `Slice<UnsafeBufferPointer<UInt8>>` need it?
+
+**TODO**: Do we do a `Sequence.withSpanIfAvailable` API?
```

Comment:
We should leave this decision to the update of the `Sequence`&`Collection` hierarchy.

Reply:
Sure, I'll update the future directions portion. What about something like `Array.withUnsafeBytes`? We could/should aggressively move people onto `RawSpan`.